### ISSUE TITLE
License page for libraries

### ISSUE THREAD

**Issue by velddev** [Mikibot/web#22]
Lets add a license page for developers. But what license do we use?

**Comment by Niputi** [comment]
accord to https://github.com/Mikibot/MikiStd/blob/master/LICENSE its mit

**Comment by velddev** [comment]
Yes but this license will be distributed for every miki lib (private ones)

**Comment by Niputi** [comment]
/licenses/mikistd
/licenses/????
and so on

**Comment by velddev** [comment]
I think we only need one. we only use one singular license everywhere
anyways

**Comment by Niputi** [comment]
no. some repos use MIT and some other use Apache license

**Comment by Fuzen-py** [comment]
> /licenses/mikistd
> /licenses/????
> and so on

I like this option, /licenses can be used to give credit to other software being used (some licenses require this).

**Comment by velddev** [comment]
I'm more worried about the logistics then, who will be responsible for
keeping these licenses up to date?

**Comment by Fuzen-py** [comment]
We should be able to just write a script to handle this

**Comment by velddev** [comment]
> /licenses can be used to give credit to other software being used (some licenses require this).

How would we do this though in a script?

**Comment by Fuzen-py** [comment]
We should probably do that part manually, or create some file in repositories that can be composed together to do this. 
A timed / master hook that:
1.  compare license checksum from repositories
3. composes some accreditation file together
4. compare checksum of the composed files
5. generate a static site (Dependent on 1|4) 
6. deploy (dependent on 5)

This makes updates dependent on the maintainer of the repositories but not requiring work in a secondary repository. As licenses update (New maintainers, contributors, another year passed) it's updated.
